search.js

```diff
@@ -0,0 +1,37 @@
+var inventory = [
+  { description: "Cat food for lazy cats.", name: "Lazy cat", price: "$10.99", image: "lazy-cat.png"},
+  { description: "Cat food for fat cats.", name: "Fat cat", price: "$11.99", image: "fat-cat.png"},
+  { description: "Cat food for angry cats.", name: "Angry cat", price: "$9.99", image: "angry-cat.png"},
+  { description: "Cat food for grumpy cats.", name: "Grumpy cat", price: "$12.99", image: "grumpy-cat.png"},
+  { description: "Cat food for skinny cats.", name: "Skinny cat", price: "$15.99", image: "skinny-cat.png"},
+  { description: "Cat food for cute cats.", name: "Cute cat", price: "$14.99", image: "cute-cat.png"},
+  { description: "Cat food for kittnes.", name: "Kittens", price: "$19.99", image: "kittens.png"}
+];
+
+function showNewInventory(inventory, theSearchTerm, newInventory) {
+  for (var i = 0 ; i < inventory.length; i++) {
+    var matchingItems = [];
```

Comment:
Separate functionality by concern.
1. Make a function for getting only the matching items.
2. Make a function that creates a DOM element for an inventory item.